jobs/integr8ly/installation-pipeline-qe-pony.yaml

```diff
@@ -0,0 +1,104 @@
+---
+
+- job:
+    name: installation-pipeline-qe-pony
+    project-type: pipeline
+    description: "Installs Integreatly remotely from Jenkins slave to QE Pony cluster."
+    sandbox: false
+    concurrent: true
```

Comment:
Since this is qe pony specific, do we want to allow concurrent builds? I would say there is no point in running install playbook twice against the same cluster in parallel.

Reply:
Yes let's remove this